### ISSUE TITLE
Spaces in site states ("plasma membrane" from Translocation) cause KaSa/KaSim to error

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#189]
Translocation statements (and possibly others?) can result in the passing through of attributes with invalid names to PySB monomers. This causes KaSa (at a minimum--probably KaSim as well) to fail. Here the Translocation statement generated by the TRIPS processor results in a Monomer with site state "plasma membrane", containing an invalid space.

```
In [3]: from indra import trips

In [12]: tp = trips.process_text('Active GEF proteins activate RAS that is at the plasma membrane.')

In [5]: from indra.assemblers.pysb_assembler import PysbAssembler

In [13]: pa = PysbAssembler()

In [14]: pa.add_statements(tp.statements)

In [15]: pa.make_model()
Out[15]: <Model 'None' (monomers: 2, rules: 1, parameters: 3, expressions: 0, compartments: 0) at 0x1230fbc50>

In [20]: pa.model.monomers
Out[20]: 
ComponentSet([
 Monomer('GEF', ['activity'], {'activity': ['inactive', 'active']}),
 Monomer('RAS', ['loc', 'activity'], {'loc': ['plasma membrane'], 'activity': ['inactive', 'active']}),
 ])

In [18]: from pysb import kappa

In [19]: im = kappa.influence_map(pa.model)
---------------------------------------------------------------------------
KasaInterfaceError                        Traceback (most recent call last)
<ipython-input-19-318ad85dfe03> in <module>()
----> 1 im = kappa.influence_map(pa.model)

/Users/johnbachman/Dropbox/1johndata/programs/pysb/pysb/kappa.py in influence_map(model, **kwargs)
    375 
    376     kasa_result = run_static_analysis(model, influence_map=True,
--> 377                                     contact_map=False, **kwargs)
    378     return kasa_result.influence_map
    379 

/Users/johnbachman/Dropbox/1johndata/programs/pysb/pysb/kappa.py in run_static_analysis(model, influence_map, contact_map, cleanup, output_prefix, output_dir, verbose)
    297     if p.returncode:
    298         err_msg = p_out + b'\n' + p_err
--> 299         raise KasaInterfaceError(err_msg.decode('utf-8'))
    300 
    301     # Try to create the graphviz objects from the .dot files created

KasaInterfaceError: Kappa Static Analyzer (DomainBased-628-g7da4236) (without Tk interface)
Analysis launched at 2017/03/08 11:25:19 (GMT-5) on SB-SORGE-1JAB69.local
Parsing /var/folders/1x/qf_k0tln39l637nj4jzrytgh0000gp/T/tmpKappa_None_40ds2wsq/None.ka...

File "/var/folders/1x/qf_k0tln39l637nj4jzrytgh0000gp/T/tmpKappa_None_40ds2wsq/None.ka", line 2, characters 23-31:
Invalid internal state
```

**Comment by bgyori** [comment]
We probably need to use the existing `_n` function to canonicalize these location names in the PysbAssembler.

**Comment by johnbachman** [comment]
Yeah that would do it.

**Comment by bgyori** [comment]
Fixed in 2d8046ef1f18b36d4103714a36c1eafc4ae65916
